### PR TITLE
debugging tooling: run `cilium bpf policy get` in debuginfo and bugtool with `-n` option

### PR DESCRIPTION
**Pull request by ianvernon**
debuginfo: get numeric identities in policymap output

This makes searching the PolicyMap for a given identity much easier than having to map labels to numeric identity. The numeric identities can be compared to the output of \`cilium identity list\` if needed.

bugtool: get all identities in policy maps by numeric identity

The non-numeric output is hard to parse, and the numeric output is much more concise.

Signed-off by: Ian Vernon <ian@cilium.io>

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/6045)

### CODE REVIEW

**Comment by ianvernon** [comment]
test-me-please

**Review by joestringer — APPROVED** [review]
